### PR TITLE
Added/Improved Intros for the Map Topics

### LINKED ISSUE #4850: Write or improve intros for getting started map topics
### What article on docs.github.com is affected?

- [ ] "[Quickstart](https://docs.github.com/en/github/getting-started-with-github/quickstart)"
- [ ] "[Learning about GitHub](https://docs.github.com/en/github/getting-started-with-github/learning-about-github)"
- [ ] "[Signing up for GitHub](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-github)"
- [ ] "[Using GitHub](https://docs.github.com/en/github/getting-started-with-github/using-github)"
- [ ] "[Exploring projects on GitHub](https://docs.github.com/en/github/getting-started-with-github/exploring-projects-on-github)"
- [ ] "[Getting started with Git](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git)"
- [ ] "[Using Git](https://docs.github.com/en/github/getting-started-with-github/using-git)"

### What part(s) of the article would you like to see updated?

These map topics (pages that group articles) either don't have intros or have intros that could be improved to be friendlier to our audience of new users. Here are some guidelines on writing good intros.

> Intros are displayed in search results and provide contextual information to help readers choose a result. Within a page, intros allow readers to quickly decide if the category, map topic, or article is relevant to them.
>
> Map topic and category intros should be one sentence long, and general and high-level enough to scale with future changes without needing to be frequently updated. Avoid lists of features or article content, and focus on the more evergreen overall task the category or map topic describes.

### Additional information
This issue is for a special project and is not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #4850 

### What's being changed:
  - Added/Updated Intros for the following Map Topics,
      - ["QuickStart"](https://docs.github.com/en/github/getting-started-with-github/quickstart)
      - ["Learning about GitHub"](https://docs.github.com/en/github/getting-started-with-github/learning-about-github)
      - ["Signing Up for GitHub"](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-github)
      - ["Using GitHub](https://docs.github.com/en/github/getting-started-with-github/using-github)
      - ["Exploring projects on GitHub"](https://docs.github.com/en/github/getting-started-with-github/exploring-projects-on-github)
      - ["Getting started with Git"](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git)
      - ["Using Git"](https://docs.github.com/en/github/getting-started-with-github/using-git)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
That's a lot of intros! Thanks for opening a PR @codewithdev 🎉 I'll get this triaged for review ✨

File: content/github/getting-started-with-github/exploring-projects-on-github/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Exploring projects on GitHub
-intro: ''
+intro: 'Discover the relevant projects on {% data variables.product.prodname_dotcom %} and make contribution to Open Source by collaborating with other people.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/exploring-projects-on-github/index.md:3]: This doesn't only apply to GitHub.com, so let's use the more general `product_name`, which will also render as GitHub Enterprise Server or GitHub AE as appropriate.

```suggestion
intro: 'Discover interesting projects on {% data variables.product.product_name %} and contribute to open source by collaborating with other people.'
```

File: content/github/getting-started-with-github/getting-started-with-git/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with Git
-intro: ''
+intro: '
+Git is a distributed Version Control System(VCS). Get started setting up Git to manage and control all your {% data variables.product.prodname_dotcom %} project workflows locally.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/getting-started-with-git/index.md:4]: Let's limit to this one sentence. I also switched "locally" with the more beginner-friendly "from your computer."

```suggestion
intro: 'Set up Git, a distributed version control system, to manage your {% data variables.product.product_name %} repositories from your computer.'
```

File: content/github/getting-started-with-github/learning-about-github/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Learning about GitHub
-intro: '{% data variables.product.prodname_dotcom %} has a variety of products and account types to help you build software and collaborate with other people.'
+intro: 'Learn more about the features and products of {% data variables.product.prodname_dotcom %} that integrates into your software management process and helps to collaborate with other people.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/learning-about-github/index.md:3]:
```suggestion
intro: 'Learn how you can use {% data variables.product.company_short %} products to improve your software management process and collaborate with other people.'
```

File: content/github/getting-started-with-github/using-git/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Using Git
-intro: ''
+intro: 'Learn about tools and terminologies used in Git to manage workflows of your remote repositories from a local machine.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/using-git/index.md:3]:
```suggestion
intro: 'Use Git to manage your {% data variables.product.product_name %} repositories from your computer.'
```

File: content/github/getting-started-with-github/quickstart/index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Quickstart
-intro: 'Quickly get started with common tasks on {% data variables.product.prodname_dotcom %}.'
+intro: 'Find and Get started working with the resources available on {% data variables.product.prodname_dotcom %}.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/quickstart/index.md:3]:
```suggestion
intro: 'Get started using {% data variables.product.product_name %} to manage Git repositories and collaborate with others.'
```

File: content/github/getting-started-with-github/signing-up-for-github/index.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Signing up for GitHub
-intro: ''
+intro: 'Sign up with your Email to Get started with setting up a new {% data variables.product.prodname_dotcom %} account.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/signing-up-for-github/index.md:3]:
```suggestion
intro: 'Start using {% data variables.product.prodname_dotcom %} for yourself or your team.'
```

File: content/github/getting-started-with-github/using-github/index.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Using GitHub
-intro: ''
+intro: '{% data variables.product.prodname_dotcom %} is accessible in multiple platforms and devices where you can explore the features and products of it.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/using-github/index.md:3]:
```suggestion
intro: 'Explore {% data variables.product.company_short %}'s products from different platforms and devices.'
```

**Review by lecoursen — DISMISSED** [review]
Thanks @codewithdev! I left some suggestions, which I'll apply now before merging. ⚡

File: content/github/getting-started-with-github/using-github/index.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Using GitHub
-intro: ''
+intro: 'Explore {% data variables.product.company_short %}'s products from different platforms and devices.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/using-github/index.md:3]:
```suggestion
intro: "Explore {% data variables.product.company_short %}'s products from different platforms and devices."
```

**Review by lecoursen — APPROVED** [review]

**Comment by codewithdev** [comment]
> Thanks @codewithdev! I left some suggestions, which I'll apply now before merging. ⚡ 

Looks great! 🎉

**Comment by codewithdev** [comment]
@lecoursen Suggestion Edits are great! ⚡

**Comment by codewithdev** [comment]
> Thanks @codewithdev! I left some suggestions, which I'll apply now before merging. ⚡

@lecoursen Suggestion Edits are great ⚡

**Pull request merged**